web/packages/shared/components/UnifiedResources/UnifiedResources.tsx

```diff
@@ -100,14 +100,32 @@ export type UnifiedResourcesPinning =
       kind: 'hidden';
     };
 
+// BulkAction is a component that allows you perform an action
```

Comment:
```suggestion
// BulkAction describes a component that allows you to perform an action
```

Reply:
also should we start commenting function/type stuff with the `/**/`? i came from another PR that converted all of an existing type comments including its fields